### PR TITLE
Adicionar read.md

### PR DESCRIPTION
**Pull request by testJoao**

### CODE REVIEW

**Comment by testJoao** [comment]
Please review this, @casey-collab

File: portugues/teste/reame.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Isto é um teste de um jogo.
```
Comment (casey-collab) [inline portugues/teste/reame.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
This looks like a test pull request to me rather than a part of the story. That's fine, but I only want to merge changes that add to the story, so I'm going to close this. If I've misunderstood, please re-open the pull request and let me know.

**Pull request closed without merging**